app/routes/socialEvents.tsx

```diff
@@ -0,0 +1,9 @@
+import EventCard from '~/components/common/eventCard';
+import { eventDetails } from '../components/common/eventCard/utils/eventDetailsObject';
+export default function socialEvents() {
+  return (
+    <>
+      <EventCard eventTitle="React India Conf" participantCount={100} eventDetails={eventDetails} />
+    </>
```

Comment:
no need of `Fragment` when you have just one child

Reply:
Done